src/libraries/System.Linq.Expressions/src/System/Linq/Expressions/IndexExpression.cs

```diff
@@ -263,9 +263,9 @@ private static string GetArgTypesString(Expression[] arguments)
             return argTypesStr.ToString();
         }
 
-        private static PropertyInfo FindProperty(Type type, string propertyName, Expression[] arguments, BindingFlags flags)
+        private static PropertyInfo? FindProperty(Type type, string? propertyName, Expression[]? arguments, BindingFlags flags)
```

Comment:
>? [](start = 67, length = 1)

Are any callers passing a name that might be `null`? If not, consider changing to `string propertyName`.

Reply:
no caller passing null, updated